sys/kern/kern_proc.c

```diff
@@ -2737,14 +2737,10 @@ kern_proc_vmmap_out(struct proc *p, struct sbuf *sb, ssize_t maxlen, int flags)
 			kve->kve_protection |= KVME_PROT_WRITE;
 		if (entry->protection & VM_PROT_EXECUTE)
 			kve->kve_protection |= KVME_PROT_EXEC;
-#ifdef OBJ_NOLOADTAGS
-		if (obj != NULL && (obj->flags & OBJ_NOLOADTAGS) == 0)
+		if (entry->protection & VM_PROT_READ_CAP)
 			kve->kve_protection |= KVME_PROT_LOADTAGS;
```

Comment:
I guess this probably wants renaming to match

Reply:
Hmmm, perhaps.  They are also currently displayed as 'ls' in procstat -v output rather than the 'RW' we've used for cap permissions.  I could do that as a followup change perhaps.

Reply:
Also `{en:load_tags/%s}`/`{en:store_tags/%s}`